ec2utils/ec2deprecateimg/ec2deprecateimg

```diff
@@ -0,0 +1,208 @@
+#DEBUG -- Local test REMOVE
+sys.path.append('/work/rjschwei/devel/Enceladus/ec2utils/ec2deprecateimg/lib')
+
+import ec2utils.ec2deprecateimg as ec2depimg
+from ec2utils.ec2DepImgExceptions import *
+
+
+# Set up command line argument parsing
+argparse = argparse.ArgumentParser(description='Deprecate images in EC2')
+argparse.add_argument(
+    '-a', '--account',
+    dest='accountName',
+    help='Account to use',
+    metavar='ACCOUNT_NAME',
+    required=True
+)
+hlpMsg = 'The deprecation period, image will be tagged for removal '
+hlpMsg += 'on "now + deprecation perion", specified in months, default '
+hlpMsg += ' is 6 month (Optional)'
+argparse.add_argument(
+    '-d', '--deprecation-period',
+    default=6,
+    dest='depTime',
+    help=hlpMsg,
+    metavar='NUMBER_OF_MONTHS',
+    type=int
+)
+hlpMsg = 'Do not perform any action, print information about actions that '
+hlpMsg += 'would be performed instared (Optional)'
+argparse.add_argument(
+    '-n', '--dry-run',
+    action='store_true',
+    default=False,
+    dest='dryRun',
+    help=hlpMsg
+)
+argparse.add_argument(
+    '-f', '--file',
+    default = os.path.expanduser('~') + os.sep + '.ec2utils.conf',
+    dest='confFile',
+    help='Path to configuration file, default ~/.ec2utils.conf (Optional)',
+    metavar='CONFIG_FILE'
+)
+depImgIdentifier = argparse.add_mutually_exclusive_group(required=True)
+depImgIdentifier.add_argument(
+    '--image-id',
+    dest='depImgID',
+    help='The AMI ID of the image to be deprecated (Optional)',
+    metavar='AMI_ID'
+)
+depImgIdentifier.add_argument(
+    '--image-name',
+    dest='depImgName',
+    help='The image name of the image to be deprecated (Optional)',
+    metavar='IMAGE_NAME'
+)
+hlpMsg = 'An image name fragment to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+depImgIdentifier.add_argument(
+    '--image-name-frag',
+    dest='depImgNameFrag',
+    help=hlpMsg,
+    metavar='IMAGE_NAME_FRAGMENT'
+)
+hlpMsg = 'A regular expression to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+depImgIdentifier.add_argument(
+    '--image-name-match',
+    dest='depImgNameMatch',
+    help=hlpMsg,
+    metavar='REGEX'
+)
+argparse.add_argument(
+    '--image-virt-type',
+    choices=['hvm','para'],
+    dest='virtType',
+    help='The virtualization type of the image to be deprecated (Optional)',
+    metavar='VIRT_TYPE'
+)
+argparse.add_argument(
+    '--public-only',
+    action='store_true',
+    default=False,
+    dest='publicOnly',
+    help='Only consider images that are public (Optional)'
+)
+replImgIdentifier = argparse.add_mutually_exclusive_group(required=True)
+hlpMsg = 'The AMI ID of the image used as a replacement for the image(s) '
+hlpMsg += 'being deprecated (Optional), one of --replacement-id or '
+hlpMsg += '--replacement-name is required'
+replImgIdentifier.add_argument(
+    '--replacement-id',
+    dest='replImgID',
+    help=hlpMsg,
+    metavar='AMI_ID'
```

Comment:
Did you intend to use the same `metavar` twice? ( https://github.com/SUSE/Enceladus/pull/8/files#diff-e6f13b1dd86ec7269495a20a7410fbbaR73 )